package.json

```diff
@@ -24,7 +24,8 @@
     "l10n:prime": "_scripts/l10n/prime.sh",
     "l10n:bundle": "_scripts/l10n/bundle.sh",
     "legal:clone": "_scripts/clone-legal-docs.sh",
-    "gql:allowlist": "nx run-many -t extract-gql && nx run-many -t copy-gql"
+    "gql:allowlist": "nx run-many -t extract-gql && nx run-many -t copy-gql",
+    "prepare": "husky install"
```

Comment:
Why does this need to be added?

Reply:
While trying to fix the husky-run error, this was recommended. But Reino found a solution. Removing now.